lib/hydra/works/models/concerns/block_child_objects.rb

```diff
@@ -0,0 +1,21 @@
+module Hydra::Works
+  # Do not allow aggregation of child objects
+  module BlockChildObjects
+
+    def child_objects= objects
+      raise NoMethodError, "undefined method `child_objects=' for #{self}"
```

Comment:
How about just StandardError "not allowed to call child_objects= on #{self.class}"